README.dev.md

```diff
@@ -4,66 +4,90 @@ Installing the Zulip Development environment
-  version of Vagrant][vagrant-dl], and then do the following:
+Ubuntu 14.04
+------------
+*  [download a newer  version of Vagrant][vagrant-dl], and then do the following:
+ 
   ```
   sudo apt-get install lxc lxc-templates cgroup-lite redir
   sudo dpkg -i vagrant*.deb # in directory where you downloaded vagrant
   vagrant plugin install vagrant-lxc
   ```
 
-* For other Linux hosts with a kernel above 3.12, [follow the Vagrant
-  LXC installation instructions][vagrant-lxc] to get Vagrant with LXC
-  for your platform.
+Other Linux (kernal above 3.12):
+--------------------------------
+* [follow the Vagrant LXC installation instructions][vagrant-lxc] to get 
+  Vagrant with LXC for your platform.
+
 
-* If your host is OS X or older Linux, [download VirtualBox][vbox-dl],
-  [download Vagrant][vagrant-dl], and install them both.
+OS X or older Linux (kernal at or below 3.12):
+----------------------------------------------
+* [download VirtualBox][vbox-dl], [download Vagrant][vagrant-dl], and install
+  them both.
 
-* If you're on OS X and have VMWare, it should be possible to patch
-  Vagrantfile to use the VMWare vagrant provider which should perform
+* Note: If you're on OS X and have VMWare, it should be possible to patch
+  Vagrantfile to use the VMWare vagrant provider, which should perform
   much better than Virtualbox.  Patches to do this by default if
   VMWare is available are welcome!
 
-* On Windows: You can use Vagrant and Virtualbox/VMWare on Windows
-  with Cygwin, similar to the Mac setup.  Be sure to create your git
-  clone using `git clone https://github.com/zulip/zulip.git -c
-  core.autocrlf=false` to avoid Windows line endings being added to
-  files (this causes weird errors).
+Windows:
+--------
+* use Vagrant and Virtualbox/VMWare on Windows with Cygwin, similar to the Mac 
+  setup. 
 
 [vagrant-dl]: https://www.vagrantup.com/downloads.html
 [vagrant-lxc]: https://github.com/fgrehm/vagrant-lxc
 [vbox-dl]: https://www.virtualbox.org/wiki/Downloads
- 
+
+Universal Vagrant instructions:
+-------------------------------
 Once that's done, simply change to your zulip directory and run
 `vagrant up` in your terminal to install the development server.  This
-will take a long time on the first run because Vagrant needs to
-download the Ubuntu Trusty base image, but later you can run `vagrant
-destroy` and then `vagrant up` again to rebuild the environment and it
-will be much faster.
+will take a long time on the first run and also fail to fully execute
+on Windows (we're working on it). Vagrant needs to download the Ubuntu
+Trusty base image, but later you can run `vagrant destroy` and then
+`vagrant up` again to rebuild the environment and it will be much faster. 
+
+On Windows,  run `npm install --no-bin-links`, and scroll to the
+"All Systems" instructions located way far below. Start from after
+`npm install`. The problem is symbolic linking is being attempted 
+on Windows and crashes `vagrant up`. 
```

Comment:
I've fixed it so it shouldn't fail on Windows with the patch to Vagrantfile. I should have squashed this commit.